fossa/analyze/action.yml

```diff
@@ -0,0 +1,40 @@
+name: fossa analyze
+
+description: Run FOSSA CLI analyze command
+
+inputs:
+  api-key:
+    description: The API key to access fossa.com
+    required: true
+  branch:
+    description: Name of the branch to be scanned
+    required: true
+  configuration-file:
+    description: Path to the FOSSA configuration file
+    required: false
+    default: .fossa.yml
+  path:
+    description: Path to the directory to be scanned
+    default: .
+    required: false
+  revision-id:
```

Comment:
i wonder if we could just default this to the current branch HEAD ?

Reply:
Depending of the type of event, HEAD may be retrieved from different attributes of the GitHub context (push vs pull-request). I prefer to keep it explicit for now, at least until the implementation of PR checks.